pages.nl/common/rm.md

```diff
@@ -0,0 +1,25 @@
+# rm
+
+> Verwijder bestanden of mappen.
+> Zie ook: `rmdir`.
+> Meer informatie: <https://www.gnu.org/software/coreutils/rm>.
+
+- Verwijder specifieke bestanden:
+
+`rm {{pad/naar/bestand1 pad/naar/bestand2 ...}}`
+
+- Verwijder specifieke bestanden, maar negeer niet-bestaanden:
```

Comment:
I am not sure what the English sentence was but what do you mean with `maar negeer niet-bestaanden:`